test/canary-environment/models/pg_cdc/wmr.sql

```diff
@@ -0,0 +1,37 @@
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Use of this software is governed by the Business Source License
+-- included in the LICENSE file at the root of this repository.
+--
+-- As of the Change Date specified in that file, in accordance with
+-- the Business Source License, use of this software will be governed
+-- by the Apache License, Version 2.0.
+
+-- TPC-H Query #Q18, chosen for the workload for its GROUP BY clause
+-- We have modified the predicate to sum(l_quantity) > 250
+-- to ensure the result updates frequently
+
+-- depends_on: {{ ref('pg_cdc') }}
+{{ config(materialized='materializedview', indexes=[{'default': True}]) }}
+
+WITH MUTUALLY RECURSIVE
+    symm (a int, b int) AS (
+        SELECT a, b FROM {{ source('pg_cdc','relationships') }}
+        UNION
+        SELECT b, a FROM {{ source('pg_cdc','relationships') }}
+    ),
+    candidates (a int, b int, degree int) AS (
+        SELECT a, b, 1 FROM symm
+        UNION
+        SELECT symm.a, reach.b, reach.degree + 1
+        FROM symm, reach
+        WHERE symm.b = reach.a
+    ),
+    reach(a int, b int, degree int) AS (
+        SELECT a , b , min(degree) FROM candidates group by a, b HAVING a != b
```

Comment:
```suggestion
        SELECT a, b, min(degree) FROM candidates group by a, b HAVING a != b
```
Nit.

Reply:
Fixed in another branch, so will not be part of this PR as I switched branches.